draft-wussler-openpgp-pqc.md

```diff
@@ -674,7 +674,7 @@ defines the instances of the following operations:
 
 and
 
-    (eccKeyShare) <- ECC-KEM.Decaps(eccSecretKey, eccCipherText)
+    (eccKeyShare) <- ECC-KEM.Decaps(eccPublicKey, eccSecretKey, eccCipherText)
```

Comment:
I don't see that the public key is needed here. Consider from the NIST IPD for ML-KEM:

`Algorithm 17 ML-KEM.Decaps(c, dk)`

This is just taking the ciphertext and the decapsulation key. The `ek_{PKE}` that is needed in the algorithm is a part of `dk`.

Reply:
```suggestion
    (eccKeyShare) <- ECC-KEM.Decaps(eccSecretKey, eccCipherText, eccPublicKey)
```

Reply:
@falko-strenzke In dilithium dk contains ek explicitly, for ECC it doesn't

Reply:
To recall, the history of the appearance of `eccPublicKey` in the final step of our ECC-KEMs, i.e. `x25519kem`, `x448kem` and `ecdhkem`, is that
- we introduced `eccKeyShare = Hash(X || eccCipherText)` (previously it was just `eccKeyShare = X`) to have IND-CCA2 secure ECC-KEMs following Cramer-Shoup. The crucial aspect of Cramer-Shoup is the inclusion of `eccCipherText`. 
- we raised this aspect security aspect during the inclusion of X25519 and X448 in the crypto-refresh and pointed the OpenPGP WG to the security considerations of RFC7748 which states "Alice and Bob can then use a key-derivation function that includes K, K_A, and K_B to derive a symmetric key.". In our terminology K_A, K_B are `eccPublicKey` and `eccCipherText`. It has been applied to the crypto-refresh such that they included both in their KDF.
- we adopted the additional inclusion of `eccPublicKey` in our ECC-KEMs to align to the approach of the crypto-refresh.
ML-KEM doas not need such an explicit construction as it does extract the public key from the decapsulation key as @falko-strenzke states it, i.e. already incorporates such an IND-CCA2 conversion and accomodates the needed data in the decapsulation key.

Reply:
> @falko-strenzke In dilithium dk contains ek explicitly, for ECC it doesn't

yes, I didn't read correctly :-(